Userland/Libraries/LibCore/Account.cpp

```diff
@@ -16,7 +16,7 @@
 #include <errno.h>
 #include <grp.h>
 #include <pwd.h>
-#ifndef AK_OS_BSD_GENERIC
+#if !defined(AK_OS_BSD_GENERIC) && !defined(AK_OS_HAIKU)
```

Comment:
Is it possibly more reasonable to add HAIKU to the list of platforms that don't compile this file in LibCore/CMakeLists.txt?

https://github.com/SerenityOS/serenity/blob/88a3b4da9e20c4a86cf6fbefb21e8a070d0731c9/Userland/Libraries/LibCore/CMakeLists.txt#L37-L44

Reply:
This would also allow dropping the Group.cpp commit

Reply:
I'm not sure if this is a good idea.
While Haiku does currently not support multiple users in its desktop environment, it does support Unix system accounts and permission and such.
Not compiling this file at all looks more like a solution for systems that don't use this type of account management and have other incompatible solutions, like Android or Windows.
In my opinion, it's better handle Haiku similar to BSD here.

Reply:
Fair enough. That is definitely true that those other platforms don't have unix users and groups at all in the standard sense